File: Dave Min/2023-05-28 Missing Number.java

class Solution {
    public int missingNumber(int[] nums) {
        boolean exist[] = new boolean[nums.length+1];
        for(var v : nums){
            exist[v] = true;
        }
        for(int i=0;i<exist.length;i++){
            if(!exist[i]) return i;
        }
        return 0;
    }
}

File: kiwoon-jung/2023-05-28-Missing-Number.js
var missingNumber = function (nums) {
  let map = new Set(nums);

  let expectedLength = nums.length + 1;

  for (let number = 0; number < expectedLength; number++) {
    if (!map.has(number)) {
      return number;
    }
  }

  return -1;
};
